ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1610,6 +1611,92 @@ private class JdbcLedgerDao(
     }
   }
 
+  private val SQL_SELECT_COMMAND = SQL(
+    """
+      |select submittedAt, ttl, success, error,
+      |from participant_command
```

Comment:
```suggestion
      |from participant_commands
```

Reply:
I'm introducing the `participant_completions` table. Would it make sense to rather call this `participant_submissions`?

Reply:
Think about it, it would probably it would make sense to prepend `command` to both to not mix them up with other submission/completion tracking tables (for parties, packages, etc.)

`participant_command_submissions` and `participant_command_completions`?

Reply:
Table renamed to `participant_command_submissions`